packaging/debian/control

```diff
@@ -29,3 +29,9 @@ Architecture: any
 Depends: ${misc:Depends}, nvidia-container-toolkit-base (= @VERSION@)
 Description: NVIDIA Container Toolkit Operator Extensions
  Provides tools for using the NVIDIA Container Toolkit with the GPU Operator
+
+Package: nvidia-container-toolkit-cdi-refresh
+Architecture: any
+Depends: ${misc:Depends}, nvidia-container-toolkit-base (= @VERSION@)
```

Comment:
This means that this will not be installed by default. Should we just include the Systemd units in the `-base` package?

Reply:
Yes, not by default. Maybe yes, it makes sense to move to the base pkg.